client/src/app/+admin/follows/followers-list/followers-list.component.scss

```diff
@@ -21,3 +21,10 @@ a {
     @include margin-right(10px);
   }
 }
+
+my-delete-button {
+  ::ng-deep .button-label {
```

Comment:
I think you should set a width on the button, that automatically uses ellipsis and does not take more width that defined by the parent. It would prevent `::ng-deep` use

Reply:
thanks for the tip !